### PR TITLE
New linter

### PR DESCRIPTION
**Pull request by gonziesc**
## Summary

- using new wolox eslint

### CODE REVIEW

File: scripts/kick-off.js [hunk 1]
```diff
@@ -36,7 +36,7 @@ const config = {
 
 const deleteFolderRecursive = path => {
   if (fs.existsSync(path)) {
-    fs.readdirSync(path).forEach((file, index) => {
+    fs.readdirSync(path).forEach((file) => {
```
Comment (arinaldi118) [inline scripts/kick-off.js]: :thinking: why doesn't the linter work for this case?  `forEach((file) =>`

Comment (gonziesc) [reply scripts/kick-off.js]: we deleted that rule

Comment (arinaldi118) [reply scripts/kick-off.js]: why? :crying_cat_face:

File: app/logger/index.js [hunk 2]
```diff
@@ -29,7 +29,9 @@ const logger = new winston.Logger({
       level: 'error',
       prettyPrint: true,
       handleExceptions: config.loggerHandlesExceptions,
+      /* eslint-disable id-length */
```
Comment (mpizzagalli) [inline app/logger/index.js]: how much is the max length?

File: app/models/book.js [hunk 3]
```diff
@@ -17,23 +17,17 @@ module.exports = (sequelize, DataTypes) => {
     }
   );
 
-  Book.associate = function(models) {};
```
Comment (mpizzagalli) [inline app/models/book.js]: why did you remove this function? isn't the index throwing and error now?

File: app/models/user.js [hunk 4]
```diff
@@ -31,25 +31,17 @@ module.exports = (sequelize, DataTypes) => {
...
-    });
-  };
-
-  User.getByUsername = username => {
-    return User.getOne({ username });
-  };
+  User.getByUsername = username => User.getOne({ username });
 
-  User.prototype.updateModel = function(props) {
+  User.prototype.updateModel = function updateModel (props) {
```
Comment (mpizzagalli) [inline app/models/user.js]: the linter forces to name all the functions?

File: migrations/migrations/20170526115728-user-create.js [hunk 5]
```diff
@@ -1,41 +1,39 @@
...
-    });
-  },
+  up: (queryInterface, Sequelize) => queryInterface.createTable('users', {
+    id: {
+      type: Sequelize.INTEGER,
+      autoIncrement: true,
+      allowNull: false,
+      primaryKey: true
+    },
+    firstName: {
```
Comment (mpizzagalli) [inline migrations/migrations/20170526115728-user-create.js]: I also prefer the database attributes in snake_case

File: scripts/kick-off.js (hunk 1 above)
Comment (gonziesc) [reply scripts/kick-off.js]: done

File: app/logger/index.js (hunk 2 above)
Comment (gonziesc) [reply app/logger/index.js]: 30 chars

File: app/models/user.js (hunk 4 above)
Comment (gonziesc) [reply app/models/user.js]: yes

File: migrations/migrations/20170526115728-user-create.js (hunk 5 above)
Comment (gonziesc) [reply migrations/migrations/20170526115728-user-create.js]: can it be in another pr????
